common/build.gradle

```diff
@@ -27,6 +27,8 @@ dependencies {
     implementation "com.fasterxml.jackson.core:jackson-databind"
     implementation "com.fasterxml.jackson.core:jackson-core"
 
+    implementation "org.springdoc:springdoc-openapi-ui:${revOpenapi}"
```

Comment:
Can you change this to `compileOnly` dependency and add it as `implementation` in the `server` module? As it stands, projects that use `conductor-common` as a dependency will get the `springdoc-openapi-ui` needlessly.

Reply:
@aravindanr I made the changes